### PR TITLE
gn-tools harvested records option

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

**Comment by pmbohm** [comment]
So harvested records can be downloaded via gn-tool scripts

File: geonetwork/gn-tool/gn3-tool.sh [hunk 1]
```diff
@@ -590,6 +601,7 @@ main() {
             -p) gn_password="$2"; shift 2;;
             -y) uuid_action="$2"; shift 2;;
             -z) group="$2"; shift 2;;
+            -a) harvest_all="$2"; shift 2;;
```
Comment (jonescc) [inline geonetwork/gn-tool/gn3-tool.sh:604]: Geared up to take a parameter?  May be confusing down the track. The presence of the -a option is enough to signal you want to export harvested records I would have thought

File: geonetwork/gn-tool/gn3-tool.sh [hunk 2]
```diff
@@ -57,7 +59,13 @@ get_all_records() {
       gn_xsrftoken_arg="--xsrftoken $gn_xsrftoken"
     fi
 
-    python ./get-uuids.py "$gn_addr/srv/eng/xml.search?fast=index&_isHarvested=n&from=1&to=2000" $gn_user_arg $gn_password_arg $gn_xsrftoken_arg
+    if [[ $harvest_all != "" ]]; then
+      harvest_all_arg="y"
```
Comment (jonescc) [inline geonetwork/gn-tool/gn3-tool.sh:63]: I think harvest_all is confusing.  harvested_records would be better

File: geonetwork/gn-tool/gn3-tool.sh (hunk 1 above)
Comment (pmbohm) [reply geonetwork/gn-tool/gn3-tool.sh:604]: I was hoping you wouldn't notice. No iv'e got to understand this bash junk

**Comment by pmbohm** [comment]
@jonescc Review changes made

File: geonetwork/gn-tool/gn-tool.sh [hunk 3]
```diff
@@ -50,7 +51,13 @@ get_all_records() {
       gn_password_arg="--password $gn_password"
     fi
 
-    python ./get-uuids.py "$gn_addr/srv/eng/xml.search?fast=index&_isHarvested=n&from=1&to=2000" $gn_user_arg $gn_password_arg
+    if [[ $harvested_records != "n" ]]; then
```
Comment (jonescc) [inline geonetwork/gn-tool/gn-tool.sh:54]: Not sure but won't this mean its now not possible to dump non-harvested records?

Comment (jonescc) [reply geonetwork/gn-tool/gn-tool.sh:54]: Actually no - I can see you set it to 'n' by default

Comment (jonescc) [reply geonetwork/gn-tool/gn-tool.sh:54]: This if statement is just confusing now and not required.

Comment (pmbohm) [reply geonetwork/gn-tool/gn-tool.sh]: Removed the if/else

**Pull request merged**